### PR TITLE
No Bug: Update swiftlint script to include M1 mac directory.

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes
Taken from https://stackoverflow.com/questions/65991508/swiftlint-cant-be-found-on-apple-silicon-xcode

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: swiftlint.sh [hunk 1]
```diff
@@ -3,8 +3,12 @@
 #
 # Runs SwiftLint to enforce style guides
 #
+# Adds support for Apple Silicon brew directory
+export PATH="$PATH:/opt/homebrew/bin"
 
-command -v swiftlint >/dev/null 2>&1 || { echo >&2 "Please install SwiftLint via Homebrew or directly from https://github.com/realm/SwiftLint"; exit 1; }
+if which swiftlint; then
+    swiftlint autocorrect && swiftlint
```
Comment (kylehickinson) [inline swiftlint.sh]: Don't think we want the autocorrect part of swiftlint ran

**Review by kylehickinson — COMMENTED** [review]
Are we changing from requiring `swiftlint` to it being optional instead?

**Comment by iccub** [comment]
added exit command, would that suffice to keep original intent of having it required?

**Comment by kylehickinson** [comment]
Yeah adding an exit on the else branch should make it identical to before

**Review by kylehickinson — APPROVED** [review]
Tested on intel machine as well, works as usual

**Pull request merged**